biom/cli/summarize_table.py

```diff
@@ -0,0 +1,99 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2011-2015, The BIOM Format Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from __future__ import division
+
+from biom.util import biom_open, compute_counts_per_sample_stats
+from biom.parse import parse_biom_table
+from numpy import std
+from operator import itemgetter
+
+
+def summarize_table(input_fp, qualitative=False, observations=False):
+    lines = []
+    with biom_open(input_fp, 'U') as biom_f:
```

Comment:
see gitter conversation about handling this: we either implement a custom parameter type in Click to convert BIOM fp to Table, or informally make sure that each `cli` function uses `biom.load_table` to do this (for consistent handling, error msgs, etc.)